routers/repo/repo.go

```diff
@@ -180,6 +180,7 @@ func CreatePost(ctx *context.Context, form auth.CreateRepoForm) {
 		return
 	}
 
+	var repo *models.Repository
```

Comment:
It seems `repo` never be used?

Reply:
It is used below in both if-else branches.

Reply:
And `err` needs to be accessible in this scope because of 
https://github.com/go-gitea/gitea/pull/9606/files#diff-52fd67c909464d3895624e92e597bf09R237